lib/pragma-grammar.ym

```diff
@@ -117,8 +117,11 @@ module_stmt
             free($2);
             cfg_args_unref(last_module_args);
             last_module_args = NULL;
-	    if (!success)
-              YYERROR;
+            if (!success)
+              {
+                lexer->tokenize_eol--;
+                YYERROR;
+              }
           }
```

Comment:
hmm maybe we should extract this as a macro? 

e.g. PRAGMA_ENTER() to increment tokenize_eol
PRAGMA_LEAVE() to decrement it.
PRAGMA_ERROR to indicate failure.

Reply:
Good idea, thanks.